dev-clean/src/main/scala/sai/llsc/EngineBase.scala

```diff
@@ -183,44 +183,49 @@ trait EngineBase extends SAIOps { self: BasicDefs with ValueDefs =>
     }
   }
 
+  def makeShadowSeq(size: Int): StaticList[Rep[Value]] = StaticList.fill(size - 1)(ShadowV())
+
   // FIXME: Alignment: CharArrayConst, ArrayConst
   // Float Type
-  // TODO: refactor
-  //     1. using flat recursion
-  //     2. code reuse with Engine-level eval?
+  // TODO: don't add padding here; Value should generate its raw data layout
```

Comment:
What does this mean? Really?

Reply:
Oh, I really mean using ValueOps's toBytes/toShadowBytes to generate raw data layout of a scalar value.